packages/client/src/search/advancedSearch/utils.ts

```diff
@@ -549,6 +549,7 @@ const getLabelForRegistrationStatus = (
     ALL: [
       RegStatus.Archived,
       RegStatus.Certified,
+      RegStatus.CorrectionRequested,
       RegStatus.DeclarationUpdated,
       RegStatus.Declared,
       RegStatus.InProgress,
```

Comment:
Could we just use all the `RegStatuses` like how we are doing [here](https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/search/advancedSearch/utils.ts#L161)? That way the same bug won't show up when we add a new `RegStatus` next time

Reply:
updated code accordingly. hope no bug will arise if RegStatus updated in future.